src/probnum/random_variables/_normal.py

```diff
@@ -257,14 +258,30 @@ def __init__(
             entropy=entropy,
         )
 
-    @cached_property
+    @property
     def cov_cholesky(self) -> _ValueType:
         """Cholesky factor :math:`L` of the covariance
         :math:`\\operatorname{Cov}(X) =LL^\\top`."""
-        if self._compute_cov_cholesky is None:
-            raise NotImplementedError
 
-        return self._compute_cov_cholesky()
+        if not self.cov_cholesky_is_precomputed:
+            self.precompute_cov_cholesky()
+        return self._cov_cholesky_values
+
+    def precompute_cov_cholesky(self, damping_factor=COV_CHOLESKY_DAMPING):
```

Comment:
the default damping parameter is the same magic constant as before, only now it is possible to change it by calling precompute_cov_cholesky manually (see tests).

Reply:
I personally prefer `precompute_*` over `compute_*`because it suggests more strongly that it is a value that is cached for later, and might also sound a bit more like something that happens in-place.

I would be willing to change the name though, if that is preferred.

Reply:
I agree